### PR TITLE
Goa 2713

### PR DESCRIPTION
**Pull request by xwatkins**
Added autocompletion for the annotation extension filter: relationship and database names

### CODE REVIEW

File: app/directives/annotationsLink.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ angular
           scope.annotationsCount = d.data.numberOfHits;
         }, function(e){
           // obsolete terms
+          console.log(e);
```
Comment (ljgarcia) [inline app/directives/annotationsLink.js]: Do we really want to keep this console?

**Comment by ljgarcia** [comment]
Just a minor comment about a console.log at annotationsLink.js

About the functionality, the autocomplete is right now case-sensitive, so if I write "uni" it will not suggest UniProt. Would it not make sense to make it case-insensitive?

File: app/directives/annotationsLink.js (hunk 1 above)
Comment (xwatkins) [inline app/directives/annotationsLink.js]: Yes in that case it's an error we want to trace in the console.

Comment (ljgarcia) [inline app/directives/annotationsLink.js]: Ok. Just take a look about functionality (case-sensitive issue)

**Comment by xwatkins** [comment]
Added case insensitivity

**Pull request merged**